src/main/webapp/script.js

```diff
@@ -0,0 +1,17 @@
+function init() {
+  new tui.Calendar('#calendar', {
+    defaultView: 'week',
+    useCreationPopup: true,
+    useDetailPopup: true,
+    disableDblClick: true,
+    disableClick: true,
+    isReadOnly: true,
+    scheduleView: ['time'],
+    taskView: false,
+    week: {
+      workweek: true,
+    },
+  });
+}
+
+init();
```

Comment:
In general, in normal/vanilla javascript, you want to avoid adding names/identifiers to the global context. In this case, window.init() is accessible. While there are minor security issues with this (third-party scripts can inspect any global context variables and/or call them), it can cause real issues when you have namespace collisions. For example, if one of the vendor libraries also declared an `init` field/function, then you would cause issues in the files because you would be overwriting each others' variable.

You can hide this function inside of the load function to deal with both any script loading issues, along with the namespace issue together:

```
document.addEventListener('load', () => {
  function init() {
    // ...
  }

  init();
});
```